schemas/session-replay/mobile/image-wireframe-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/mobile/image-wireframe-schema.json",
+  "title": "ImageWireframe",
+  "type": "object",
+  "description": "Schema of all properties of a ImageWireframe.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-schema.json"
+    },
+    {
+      "required": ["type", "base64", "imageFileExtension"],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "image",
+          "readOnly": true
+        },
+        "base64": {
+          "type": ["string", "null"],
```

Comment:
Is there a different way we could describe this type? I know it won't be the same but making `base64` not required will make it optional. We could be sending `null` as convention on this field (after all this is temporary measure).

I'm asking because this definition is a new concept to the entire schema. Not sure Android side (@mariusc83 ?), but iOS code generator fails on handling it:
```
🛑 Decoding [CodingKeys(stringValue: "allOf", intValue: nil), _JSONKey(stringValue: "Index 1", intValue: 1), CodingKeys(stringValue: "properties", intValue: nil), _JSONKey(stringValue: "base64", intValue: nil)] is not supported in `JSONSchema.init(from:)`.
→ coding path: allOf → 1 → properties → base64 → type
```
That means, we won't be able to generate the code without first enhancing generator. If it's possible to describe it differently, we will save at least few days iOS side (such type duality is far from Swift idioms - perhaps would need to abstract it with enums).

Reply:
Yes I wanted to say the same thing, it will bring some new changes into our generator that I would gladly want to avoid if possible. I agree with @ncreated proposal to have it as an optional.

Reply:
I think it looks like an implementation constraint but honestly I don't mind going for:
- base64 not requried
- base64 type:string

so if you dont have it, it's not there and then you send a mutation that sets it.

I am not fan of the non-required and `"type": ["string", "null"],` as it's not clear how we differentiate the missing field case VS the null case

Reply:
Note: I will go for the same approach for the mimeType as I guess you will set both fields once the base64 is computed